robot_smach_states/src/robot_smach_states/navigation/follow_operator2_0.py

```diff
@@ -6,6 +6,7 @@
 import math
 import random
 import sys
+import hmi
```

Comment:
I am not reviewing following2.0

Reply:
Since this has no priority this review is left out for now.